### ISSUE TITLE
[REQUEST] Profiles for different cars

### ISSUE THREAD

**Issue by Freezli** [agronick/aa-torque#129]
Is it possible for you to add profiles so that you can create different dashboards for several cars as with Torque Pro and select them on the head unit? 

Or is it possible for AA-Torque to select the profile using the OBD adapter or to adopt the selected profile from Torque?

Now I simply create the necessary dashboards on other boards, not the best solution.

**Comment by agronick** [comment]
Yeah, I just started working on this again. That would be a great feature. I just don't have the time.
